physutils/tests/test_tasks.py

```diff
@@ -0,0 +1,107 @@
+"""Tests for physutils.tasks and their integration."""
+
+import os
+
+import physutils.tasks as tasks
+from physutils import physio
+from physutils.tests.utils import create_random_bids_structure
+
+
+def test_generate_physio_phys_file():
+    """Test generate_physio task."""
+    physio_file = os.path.abspath("physutils/tests/data/ECG.phys")
+    task = tasks.generate_physio(input_file=physio_file, mode="physio")
+    assert task.inputs.input_file == physio_file
+    assert task.inputs.mode == "physio"
+    assert task.inputs.fs is None
+
+    task()
```

Comment:
Why is `task` called here ? Physio objects are not callable

Reply:
This is how a pydra task is run after you specify the inputs. It's not a Physio object